### PR TITLE
site: fix code-box style with Badge

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

beore:

image

after:

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 无需纳入ChangeLog |
| 🇨🇳 Chinese | 无需纳入ChangeLog |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by Wxh16144** [comment]
修复前后在 pr 描述中最好贴个图

**Comment by li-jia-nan** [comment]
> 修复前后在 pr 描述中最好贴个图

image

File: .dumi/theme/common/styles/Demo.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const GlobalDemoStyles: React.FC = () => {
         .code-box {
           position: relative;
           display: inline-block;
-          width: 100%;
+          width: calc(100% - 2px);
```
Comment (MadCcc) [inline .dumi/theme/common/styles/Demo.tsx:25]: 用 token

Comment (li-jia-nan) [reply .dumi/theme/common/styles/Demo.tsx]: done

File: .dumi/theme/common/styles/Demo.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const GlobalDemoStyles: React.FC = () => {
         .code-box {
           position: relative;
           display: inline-block;
-          width: 100%;
+          width: calc(100% - ${token.lineWidth}px);
```
Comment (MadCcc) [inline .dumi/theme/common/styles/Demo.tsx:25]:
```suggestion
          width: calc(100% - ${token.lineWidth * 2}px);
```

**Review by kiner-tang — APPROVED** [review]

**Pull request merged**